tests/slurm-tests/qwen3_4b_evals/run_test.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from nemo_skills.pipeline.cli import eval, prepare_data, run_cmd, wrap_arguments
+
+
+def setup(workspace, cluster, expname_prefix):
+    # download models
+    model = "Qwen/Qwen3-4B"
+    cmd = f"huggingface-cli download {model}"
+    run_cmd(
+        ctx=wrap_arguments(cmd),
+        cluster=cluster,
+        expname=f"{expname_prefix}-download-models",
+        log_dir=f"{workspace}/download-assets",
+    )
+
+
+def eval_reasoning_on(workspace, cluster, expname_prefix, wandb_project):
+    """Run evals in Reasoning ON mode"""
+    base_model = "Qwen/Qwen3-4B"
+
+    # Common settings for reasoning ON
+    common_params = "++inference.temperature=0.6 ++inference.top_p=0.95 "
+    tokens_to_generate = "++inference.tokens_to_generate=8192 "
+
+    # BFCL (Reasoning ON)
+    eval(
+        ctx=wrap_arguments(f"{common_params} {tokens_to_generate}"),
+        cluster=cluster,
+        benchmarks="bfcl_v3",
+        model=base_model,
+        server_gpus=8,
```

Comment:
8 gpus is probably quite inefficient for such a small model, maybe 1 or 2 is better. As long as it finishes in 4 hours